apps/game_backend/lib/game_backend/users/user.ex

```diff
@@ -29,7 +29,7 @@ defmodule GameBackend.Users.User do
     has_many(:currencies, UserCurrency)
     has_many(:units, Unit, preload_order: [desc: :level])
     has_many(:items, Item)
-    has_many(:daily_quests, DailyQuest)
+    has_many(:daily_quests, UserQuest)
```

Comment:
the field name is still daily_quests, is that ok?

Reply:
nope it's not